src/components/navigation_bar.js

```diff
@@ -0,0 +1,32 @@
+import React, {Component} from 'react';
+import { Nav, Navbar} from 'react-bootstrap';
+import logo from "../images/default.png"
+import "./layout.css"
+class Navigation_Bar extends Component {
+  render() {
+    return (
+        <Navbar bg="#FFFFFF" variant="light" expand='lg'>
+            <Navbar.Brand href="#">
+            <img
+                src={logo}
+                width="300"
+                height="50"
+                className="d-inline-block align-top"
+                alt=""
```

Comment:
The `alt` (which stands for `alternate text`) needs to be filled up with some text that describes the image. This is important in case the image cannot be displayed for some reason (network connection issues) or if the reader is using a screen reader (say a visually impaired user).